vault/resource_ssh_secret_backend_role.go

```diff
@@ -363,10 +372,12 @@ func sshSecretBackendRoleRead(d *schema.ResourceData, meta interface{}) error {
 		"allowed_critical_options", "allowed_domains",
 		"cidr_list", "allowed_extensions", "default_extensions",
 		"default_critical_options", "allowed_users_template",
-		"allowed_users", "default_user", "key_id_format",
-		"max_ttl", "ttl", "algorithm_signer",
+		"allowed_users", "default_user_template", "default_user",
+		"key_id_format", "max_ttl", "ttl", "algorithm_signer",
 	}
 
+	// Some fields cannot be read from the API
```

Comment:
nit: wondering if it's worth documenting which specific API fields are not returned by Vault on read. Would also inform us which specific API fields cannot be imported

Reply:
is there precedent for this that I could take a look at for reference?

Reply:
@fairclothjm leave a comment on specific fields that are not returned